Resources/Prototypes/Entities/Objects/Fun/Tabletop/nanopoliafigurines.yml

```diff
@@ -0,0 +1,913 @@
+- type: entity
+  id: NanapoliaTabletopPiece
+  parent: BaseTabletopPiece
+  abstract: true
+  components:
+  - type: Sprite
+    noRot: true
+    sprite: Objects/Fun/Tabletop/nanofigure.rsi
+
+- type: entity
+  id: FigurineCat
+  name: Cat
+  parent: NanapoliaTabletopPiece
+  components:
+  - type: Sprite
+    state: cat
+    
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineLaceups
+  name: Laceups
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: laceups
+    
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineMcgriff
+  name: Mcgriff
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: mcgriff
+
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineTopHat
+  name: Top Hat
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: top_Hat
+
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineGreenAutolathe
+  name: Figurine Green Autolat
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: autolat_figure
+
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineRedProtolathe
+  name: Figurine Red Protolat
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: protolat_figure
+
+- type: entity
+  parent: BaseItem
+  id: FigurineGreenAutolatheBox
+  name: Spare Figurine Green Autolat
+  description: A figurine from nanopolia
+  abstract: false
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/nanofigure.rsi
+    state: autolat_figure
+
+- type: entity
+  parent: BaseItem
+  id: FigurineRedProtolatheBox
+  name: Spare Figurine Red Protolat
+  description: A figurine from nanopolia
+  abstract: false
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/nanofigure.rsi
+    state: protolat_figure
+    
+
+#Box----------------------------------------------------------------------------
+
+- type: entity
```

Comment:
i didnt see these since they were in figurines file but there are a lot of typos in these descriptions below here, give them all a proofread

Reply:
Fixed(sorry for my English)